app/renderer/js/utils/logger-util.js

```diff
@@ -5,14 +5,29 @@ const { initSetUp } = require('./default-util');
 const { sentryInit, captureException } = require('./sentry-util');
 
 initSetUp();
-sentryInit();
+
 let app = null;
 if (process.type === 'renderer') {
 	app = require('electron').remote.app;
 } else {
 	app = require('electron').app;
 }
 
+// Report Errors to Sentry only if it is enabled in settings
+// Gets the value of reportErrors from config-util for renderer process
+// For main process, sentryInit() is handled in index.js
+let reportErrors = true;
+if (process.type === 'renderer') {
```

Comment:
We are already using this condition above. So maybe wrap it up there only.

Reply:
Ok sure, will do.

Reply:
@akashnimare Made the change. Thanks.